var/spack/repos/builtin/packages/python/package.py

```diff
@@ -126,6 +126,8 @@ class Python(AutotoolsPackage):
     depends_on('tcl', when='+tkinter')
     depends_on('tix', when='+tix')
     depends_on('libuuid', when='+uuid')
```

Comment:
Don't you also need to delete this line?

Reply:
Derp, yes. I'd rebased this commit and missed the duplication when checking the diff. Good catch. Fixed.